manubot/cite/isbn.py

```diff
@@ -3,6 +3,34 @@
 import logging
 import re
 
+from .handlers import Handler
+
+
+class Handler_ISBN(Handler):
+
+    standard_prefix = "isbn"
+
+    prefixes = [
+        "isbn",
+    ]
+
+    @classmethod
```

Comment:
Is there a reason for using `@classmethod` but not in the other handlers?

Reply:
Other cases of inspect use `self` but this doesn't. Will remove `@classmethod` since it can add some confusion